src/kili/adapters/kili_api_gateway/helpers/queries.py

```diff
@@ -45,22 +34,20 @@ def execute_query_from_paginated_call(
         where: Dict[str, Any],
         options: QueryOptions,
         tqdm_desc: str,
-        nb_elements_to_query: Optional[int],
-        post_call_function: Optional[Callable[[List], List]] = None,
+        count_query: Optional[str],
```

Comment:
I would keep nb_elements_to_query

not all queries have a count resolver (dataConnections does not have countdataconnections)

Reply:
but as discussed, I'm ok to keep self.get_number_of_elements_to_query in the class, totally makes sense, but we can't use it for all queries

Reply:
If I do a method to call before to call the execute_query_from_paginated_call, we might forget to call it.
Here it is a it was before in the previous code, if we provide a count_query if it exists, it retrieve the number of elements to call, if not, it will disable tqdm. 
For methods without a count query, we just pass a count_query set as None

Reply:
ok 👍